SignInCheckIn/SignInCheckIn/Controllers/ChildSigninController.cs

```diff
@@ -226,11 +224,12 @@ public IHttpActionResult PrintParticipant(int eventParticipantId)
         [Route("signin/event/{eventId}/room/{roomId}/reverse/{eventparticipantId}")]
         public IHttpActionResult ReverseSignin(int eventId, int roomId, int eventparticipantId)
         {
-            return Authorized(token =>
+            return Authorized(authDto =>
             {
                 try
                 {
-                    var reverseSuccess = _childSigninService.ReverseSignin(token, eventparticipantId);
+                    if (!authDto.Authorization.MpRoles.ContainsKey(_kidsClubTools)) throw new UnauthorizedAccessException();
```

Comment:
Logic executed after an if statement should be in a code block, unless it's a ternary operator -

if (condition)
{
// do something
}